src/libs/wcautil/qtexec.cpp

```diff
@@ -136,76 +137,79 @@ static HRESULT LogOutput(
             ExitOnLastError(hr, "Failed to read from handle.");
         }
 
-        // Check for UNICODE or ANSI output
-        if (bFirst)
+        if (fLogOutput)
         {
-            if ((isgraph(pBuffer[0]) && isgraph(pBuffer[1])) ||
-                (isgraph(pBuffer[0]) && isspace(pBuffer[1])) ||
-                (isspace(pBuffer[0]) && isgraph(pBuffer[1])) ||
-                (isspace(pBuffer[0]) && isspace(pBuffer[1])))
+            // Check for UNICODE or ANSI output
+            if (bFirst)
             {
-                bUnicode = FALSE;
+                if ((isgraph(pBuffer[0]) && isgraph(pBuffer[1])) ||
+                    (isgraph(pBuffer[0]) && isspace(pBuffer[1])) ||
+                    (isspace(pBuffer[0]) && isgraph(pBuffer[1])) ||
+                    (isspace(pBuffer[0]) && isspace(pBuffer[1])))
+                {
+                    bUnicode = FALSE;
+                }
+
+                bFirst = FALSE;
             }
 
-            bFirst = FALSE;
-        }
-
-        // Keep track of output
-        if (bUnicode)
-        {
-            hr = StrAllocConcat(&szLog, (LPCWSTR)pBuffer, 0);
-            ExitOnFailure(hr, "failed to concatenate output strings");
-        }
-        else
-        {
-            hr = StrAllocStringAnsi(&szTemp, (LPCSTR)pBuffer, 0, CP_OEMCP);
-            ExitOnFailure(hr, "failed to allocate output string");
-            hr = StrAllocConcat(&szLog, szTemp, 0);
-            ExitOnFailure(hr, "failed to concatenate output strings");
-        }
-
-        // Log each line of the output
-        pNext = szLog;
-        pEnd = wcschr(szLog, L'\r');
-        if (NULL == pEnd)
-        {
-            pEnd = wcschr(szLog, L'\n');
-        }
-        while (pEnd && *pEnd)
-        {
-            // Find beginning of next line
-            pEnd[0] = 0;
-            ++pEnd;
-            if ((pEnd[0] == L'\r') || (pEnd[0] == L'\n'))
+            // Keep track of output
+            if (bUnicode)
             {
-                ++pEnd;
+                hr = StrAllocConcat(&szLog, (LPCWSTR)pBuffer, 0);
+                ExitOnFailure(hr, "failed to concatenate output strings");
+            }
+            else
+            {
+                hr = StrAllocStringAnsi(&szTemp, (LPCSTR)pBuffer, 0, CP_OEMCP);
+                ExitOnFailure(hr, "failed to allocate output string");
```

Comment:
Since you're fixing up the code, may as well capitalize "failed" as new code does.